src/lib/index.ts

```diff
@@ -142,6 +142,26 @@ class TPAStylePlugin {
     return escapedContent.substring(1, escapedContent.length - 1);
   }
 
+  private getStyleParamsDefaultsContent(assets: any, fileName: string) {
+    const styleParamsFileName = getRelatedStyleParamsFileName(fileName);
+    const styleParamsFile = assets[styleParamsFileName];
+
+    if (styleParamsFile) {
+      return `(() => {
```

Comment:
Not sure why do we have mixed ES6 and ES5 syntax here

Reply:
Will fix on #35